doc/changes/latest.inc

```diff
@@ -150,6 +150,8 @@ Bugs
 
 - In :class:`mne.Report`, some figures would have an undesired border added to the edges; this has now been resolved (:gh:`10730` by `Richard Höchenberger`_)
 
+- Automatically selects EEG channels when plotting bridged electrodes with :func:`mne.viz.plot_bridged_electrodes` (:gh:`10753` by `Mathieu Scheltienne`_)
```

Comment:
But what was the bug? This entry reads like an enhancement, not a bug fix

Reply:
You are right, it can be described as both, I changed it to keep it in the bug fix section.